README.md

```diff
@@ -65,8 +65,9 @@ curl -s https://raw.githubusercontent.com/PacificBiosciences/pbsv/master/annotat
 
 6) Download all `.fastq` files:
 ```sh
-curl -s ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/HG002_NA24385_son/PacBio_CCS_15kb/HG002.15kb.Q20.tar.gz > fastqs/HG002.15kb.Q20.tar.gz
-tar xvzfC fastqs/HG002.15kb.Q20.tar.gz fastqs/
+FTPDIR=ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/data/AshkenazimTrio/HG002_NA24385_son/PacBio_CCS_15kb/
+curl -s ${FTPDIR} | awk '{print $9}' | grep '.fastq' > fastqs/filelist
+for i in `less fastqs/filelist`; do curl -s ${FTPDIR}${i} > fastqs/${i}; done
```

Comment:
I recommend to: 1) use the `-l` option to curl; 2) use `$` in the grep regexp to skip the md5 files; and 3) avoid creating the intermediate `fastqs/filelist`.

```suggestion
for fastq in $(curl -s -l ${FTPDIR} | grep -E '.fastq$'); do curl -s ${FTPDIR}${fastq} > fastqs/${fastq}; done
```